guides/tactics/convoy.md

```diff
@@ -0,0 +1,98 @@
+# Convoys
+
+## Required Reading
+
+This document assumes the reader is already familiar with [Driving Roles](guides/roles/driver.md).
+
+## Operating
+
+### Speed
+There are 3 main speeds in a convoy: *Full* - 80 KM/h  |  *Half* - 40 KM/h  |  *City* - 25 KM/h  
```

Comment:
```suggestion
There are 3 main speeds in a convoy: *Full* - 80 km/h  |  *Half* - 40 km/h  |  *City* - 25 km/h  
```
Typically KM/h is not capitalised and such capitalisation should only apply to MPH.